cmd/alias.rb

```diff
@@ -16,7 +16,7 @@ def alias_args
       switch "--edit",
              description: "Edit aliases in a text editor. Either one or all aliases may be opened at once." \
                         " If the given alias doesn't exist it'll be pre-populated with a template."
-      max_named 1
+      named_args(max: 1)
```

Comment:
```suggestion
      named_args max: 1
```

Not technically needed but it's more consistent with the rest of the `brew` commands.

Also, can you update lines 11-15 to be:

```ruby
      usage_banner "`alias` [<alias> ... | <alias>=<command>]"
      description <<~EOS
        Show existing aliases. If no aliases are given, print the whole list.
      EOS
```